modules/globebrowsing/globebrowsingmodule_lua.inl

```diff
@@ -370,33 +371,81 @@ getLocalPositionFromGeo(std::string globeIdentifier, double latitude, double lon
 
 /**
  * Get geographic coordinates of the camera position in latitude, longitude, and altitude
- * (degrees and meters).
+ * (degrees and meters). If the optional bool paramater is specified, the camera 
+ * eye postion will be used instead
  */
-[[codegen::luawrap]] std::tuple<double, double, double> getGeoPositionForCamera() {
+[[codegen::luawrap]] std::tuple<double, double, double> 
+getGeoPositionForCamera(std::optional<bool> useEyePosition) 
```

Comment:
Can do `bool useEyePosition = false` instead... Shorter, more concise, and the documentation contains the default value that way